### PR TITLE
Update readme - py3.10, faster tests

### PR DESCRIPTION
**Pull request by MrCreosote**
Missed updating the readme when I bumped python previously

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ have no tests at all, and so the actual coverage of this repo is much lower than
 
 ## Setup
 
-Install Python 3.7, preferably using pyenv: https://github.com/pyenv/pyenv
+Install Python 3.10, optionally using pyenv: https://github.com/pyenv/pyenv
```
Comment (MrCreosote) [inline README.md:19]: suggest, don't editorialize

File: README.md [hunk 2]
```diff
@@ -26,14 +26,30 @@ pipenv sync --dev
 
 ## Running tests
 
-To run tests, arangodb must be running locally on the default port with default root credentials. You can run arangodb using docker-compose with `docker-compse up`.
+To run tests, `arangodb` must be running locally on the default port with default root credentials.
+You can run `arangodb` using docker-compose with `docker-compse up`. However, we have found that
+running `arangodb` outside the docker container is ~10x faster for reasons that are currently
```
Comment (dauglyon) [inline README.md:31]: On Mac at least, this is likely to do with reads/writes being slowish with the docker desktop VM

**Review by dauglyon — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (MrCreosote) [reply README.md:31]: I'm on ubuntu *shrug*

Comment (MrCreosote) [reply README.md:31]: Although to be specific I'm running an ubuntu VM in an ubuntu host. Maybe the VM + docker combination slows things down for some reason? Although I don't recall anything like this when I've used mongo in docker containers for tests in other repos

**Pull request merged**